cluster/images/conformance/Makefile

```diff
@@ -37,6 +37,8 @@ TEMP_DIR:=$(shell mktemp -d -t conformanceXXXXXX)
 all: build
 
 build:
+# This option is for running docker build with --platform flag
+export DOCKER_CLI_EXPERIMENTAL := enabled
```

Comment:
move this up before the targets.
I'm surprised the unindented ifndef below is valid..